lib/brains/webhooks.py

```diff
@@ -228,9 +227,13 @@ def update_transactions(self):
                     status=our_status,
                     status_reason=reason,
                     type=constants.TYPE_PAYMENT,
-                    uid_support=their_transaction.id,
-                    uuid=str(uuid.uuid4())
+                    uid_support=their_transaction.id
                 )
+
+                # Make the transaction short and helpful.
+                our_transaction.uuid = 'bt-' + shorter(our_transaction.pk)
+                our_transaction.save()
```

Comment:
it's a little unfortunate that we have to hit the db twice here but it's only a webhook so probably not a big deal.